plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -184,32 +184,24 @@ public static void installAllAppEngineRuntimes(IFacetedProject project, IProgres
     }
   }
 
-  public static org.eclipse.wst.server.core.IRuntime createAppEngineServerRuntime(IProgressMonitor monitor)
-      throws CoreException {
+  public static org.eclipse.wst.server.core.IRuntime createAppEngineServerRuntime(
+      IProgressMonitor monitor) throws CoreException {
     IRuntimeType appEngineRuntimeType =
         ServerCore.findRuntimeType(AppEngineStandardFacet.DEFAULT_RUNTIME_ID);
     if (appEngineRuntimeType == null) {
-      throw new NullPointerException("Could not find " + AppEngineStandardFacet.DEFAULT_RUNTIME_NAME + " runtime type");
+      throw new NullPointerException(
+          "Could not find " + AppEngineStandardFacet.DEFAULT_RUNTIME_NAME + " runtime type");
     }
 
     IRuntimeWorkingCopy appEngineRuntimeWorkingCopy
```

Comment:
The runtime definition is actually in [`.localserver`](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml#L51) and it uses the [`ServletClasspathProvider`](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ServletClasspathProvider.java) to obtain the necessary jars from Maven Central.